lib/extras/encode_jpeg.cc

```diff
@@ -75,7 +76,9 @@ void ComputeDCTCoefficients(const Image3F& opsin, const ImageF& qf,
             block[ix * 8 + iy] = cc;
           }
         }
-        block[0] = std::round((2040 * dct[0] - 1024) * qmc[0]);
+        if (xyb) {
+          block[0] = std::round((2040 * dct[0] - 1024) * qmc[0]);
```

Comment:
please add a comment explaining the different behavior, something like "ToXYB does not create zero-centered sample values like RgbToYcbcr does, so we apply an offset to the DC values instead"

Reply:
Thanks, done.